src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -262,7 +262,7 @@ private function fixPhpDoc(Tokens $tokens, int $index, array $uses, string $name
     {
         $phpDoc = $tokens[$index];
         $phpDocContent = $phpDoc->getContent();
-        Preg::matchAll('#@([^\s]+)\s+([^\s]+)#', $phpDocContent, $matches);
+        Preg::matchAll('#@([^\s]+)\s+([^\s}]+)#', $phpDocContent, $matches);
```

Comment:
Regex was changed in #7622 to match only class names, did you try if it works in your case?

Reply:
it does not

Reply:
ah, there is new capturing group there :D

Reply:
Yes, whitespace between tag and value is caught and used during replacing. IMHO these 2 PRs are in conflict, not only because of capturing groups but also part covering tags' value.

Reply:
I merged both PRs locally, there is no logic conflict, both new test cases are working properly.
I updated your PR already to incorporate changes from master after this PR got merged and your PR is green.